packages/suite-desktop/src-electron/modules/shortcuts.ts

```diff
@@ -25,6 +25,14 @@ const init: Module = ({ mainWindow }) => {
             mainWindow.webContents.reload();
         });
     });
+
+    const hardReloadAppShortcuts = ['Shift+F5', 'CommandOrControl+Shift+R'];
+    hardReloadAppShortcuts.forEach(shortcut => {
+        electronLocalshortcut.register(mainWindow, shortcut, () => {
+            logger.info('shortcuts', `${shortcut} pressed to hard reload app`);
+            mainWindow.webContents.reload();
```

Comment:
I can't see the difference in this hard reload from "regular" reload - both call `mainWindow.webContents.reload()`

Reply:
🤯 
0b02675d6c7e4da938ebaae8d1485bce75abf59c